README.md

```diff
@@ -172,6 +172,35 @@ For example:
 ┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
 ```
 
+#### Scoped Files 
+
+We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.
+
+For example:
+
+```sh
+┣ 📂 pages \
+┣ ┣ 📂 Home \
+┣ ┃ ┣ 📂 utils \
+┣ ┃ ┣ ┣ 📜 someUtils.js
+┣ ┃ ┣ 📂 helpers \
+┣ ┣ ┣ ┣ 📜 someHelper.js
+┣ ┃ ┣ 📂 hooks \
+┣ ┣ ┣ ┣ 📜 useSomeHook.js
+```
+
+If we need use these this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Comment:
```suggestion
If we need use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

"this" is still there. 😆 
[_suspense song intensifies_]

Reply:
:hehe:

Reply:
done :sai-pra-la-bug: